webpack.config.js

```diff
@@ -14,174 +14,131 @@
  */
 const path = require('path')
 const webpack = require('webpack');
-const OptimizeCssAssetsPlugin = require('optimize-css-assets-webpack-plugin')
-const MiniCssExtractPlugin = require('mini-css-extract-plugin')
 
-const DeleteAfterBuildPlugin = require('./js/webpack-plugins/delete-after-build-plugin.js')
+const MiniCssExtractPlugin = require("mini-css-extract-plugin"); //extracts css from js into css
+const CssMinimizerPlugin = require("css-minimizer-webpack-plugin");
+
+//const DeleteAfterBuildPlugin = require('./js/webpack-plugins/delete-after-build-plugin.js')
```

Comment:
The reason this plugin fails after the update to webpack 5 is that they introduced a new hook system for plugins. Luckily, the fix should be relatively simple:

In `js/webpack-plugins/delete-after-build-plugin.js`, replace:

```
compiler.plugin('done', (compilation) => {
  // Code which deletes the specified js files here
})
```

with:
```
compiler.hooks.done.tap('DeleteAfterBuildPlugin', stats => {
  // Code which deletes the specified js files here
})
```

Give that a shot, then try uncommenting this line and the lines below where the plugin is instantiated. If you're interesting in learning more about the webpack plugin system, the docs for the `done` hook [can be found here](https://webpack.js.org/api/compiler-hooks/#done), and a general guide on writing plugins [can be found here](https://webpack.js.org/contribute/writing-a-plugin).